src/localization/localization.ts

```diff
@@ -0,0 +1,47 @@
+import i18n, {Callback, InitOptions} from 'i18next';
+import {initReactI18next} from 'react-i18next';
+
+import {storeApi} from '../services/storage/storeApi';
+import {AsyncKey} from '../services/storage/asyncKey';
+import en from './translations/en.json';
+import ua from './translations/ua.json';
+import ru from './translations/ru.json';
+
+export enum Languages {
+  english = 'en',
+  ukrainian = 'ua',
+  russian = 'ru',
+}
+
+class Localization {
+  public changeLanguage = async (key: Languages) =>
+    await i18n.changeLanguage(key);
+
+  public translate = (textKey: string) => {
+    const translation = i18n.t(textKey);
+    return translation as string;
+  };
+
+  public initialization = async (options: InitOptions, callback?: Callback) => {
+    await i18n.use(initReactI18next).init({...options}, callback);
+  };
+
+  public InitLanguage = async (language: Languages) =>
+    await localization.initialization({
```

Comment:
May be this.initialization?
And what is the reason to have 2 methods that are the same ?